main.py

```diff
@@ -16,4 +16,4 @@
         snowman(snowman_word)
     else: 
       from game_test import *
-      unittest.main(exit=False)
\ No newline at end of file
+    unittest.main(exit=False)
```

Comment:
This small stray edit cause the tests to always try to run even if the player picked "p" to play. Unindenting the line moves it out of the `else` and into code that runs after the whole condition (whether it was true or false). Watch out for stray changes. If we accidentally change a file, as long as we haven't staged it yet, we can revert it with `git restore main.py`, where `main.py` is the name of the file to revert the changes of.